src/transformers/models/vipllava/modeling_vipllava.py

```diff
@@ -441,10 +441,10 @@ def forward(
                 if past_key_values is not None and pixel_values is not None and input_ids.shape[1] == 1:
                     # Retrieve the first layer to inspect the logits and mask out the hidden states
                     # that are set to 0
-                    first_layer_past_key_value = past_key_values[0][0][:, 0, :, :]
+                    first_layer_past_key_value = past_key_values[0][0][:, :, :, 0]
```

Comment:
This seems like an opportunity for `# Copied from` (not to be fixed in this PR, but in the future)

Reply:
Noted for the next PR :)